src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -418,7 +420,7 @@ public void testDestroyCacheWithWrongFileNameLength() throws Exception {
     }
 
     @Test
-    public void testCacheFile() throws Exception {
+    public void testCacheFileWithoutExternalResources() throws Exception {
```

Comment:
The name of the test means that we test caching functionality for the check which does not hold external resources.